### ISSUE TITLE
Header. Target size too small in header buttons

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#72]
### Description of issue

Target size for most header and footer links do not meet the require 44px by 44px minimum size.

In these images, each yellow square represents 44px squares. We can see that they are overlapping, which indicates a failure.

![image](https://user-images.githubusercontent.com/5230720/136087139-2f2a4d58-bf35-4341-9f96-eb1e50d885d1.png)
![image](https://user-images.githubusercontent.com/5230720/136086613-eb1cbb10-21c3-4793-b91f-ae7687d3c128.png)

### Potential fix
- having a minimum width and height of 44px by 44px on those links would fix the overlapping and make sure the target sizes are large enough.

### Resources
https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html
